ui/src/components/credentials/IssueCredentialForm.tsx

```diff
@@ -213,182 +215,266 @@ export function IssueCredentialForm({
     [env, fetchJsonSchema, initialValues.schemaID, messageAPI]
   );
 
+  const computeFormObjectInitialValues = (
+    objectAttribute: ObjectAttribute,
```

Comment:
Shouldn't this be `objectAttribute?: ObjectAttribute`?

Reply:
mmm.. why?

Reply:
Because when you call this in the same file, `credentialSubjectAttributeWithoutId` can be `undefined`:

```ts
const credentialSubjectAttributeWithoutId = isAsyncTaskDataAvailable(jsonSchema)
  ? extractCredentialSubjectAttributeWithoutId(jsonSchema.data)
  : undefined;

useEffect(() => {
  const initialValuesWithSchemaValues: Store = credentialSubjectAttributeWithoutId
    ? {
        ...initialValues,
        credentialSubject: computeFormObjectInitialValues(
          credentialSubjectAttributeWithoutId,
          initialValues.credentialSubject || {}
        ),
      }
    : initialValues;
  form.setFieldsValue(initialValuesWithSchemaValues);
}, [computeFormObjectInitialValues, credentialSubjectAttributeWithoutId, form, initialValues]);
```

Reply:
Nope, it can't. When `credentialSubjectAttributeWithoutId` is `undefined` we do not call `computeFormObjectInitialValues`. Doing so would raise a compile error!